zebra-test/src/command.rs

```diff
@@ -241,6 +241,37 @@ impl TestOutput {
         .with_section(stdout)
     }
 
+    pub fn stdout_equals(&self, s: &str) -> Result<&Self> {
+        let stdout = String::from_utf8_lossy(self.output.stdout.as_slice());
+
+        if stdout.eq(s) {
```

Comment:
I think we should prefer using the operator instead of the Trait fn for equality

```suggestion
        if stdout == s {
```